backend/EasySpeak.Core/EasySpeak.Core.BLL/Services/LessonsService.cs

```diff
@@ -16,13 +16,22 @@ public LessonsService(EasySpeakCoreContext context, IMapper mapper) : base(conte
     {
     }
 
+    public async Task<ICollection<QuestionForLessonDto>> GetQuestionsByLessonIdAsync(int id)
+    {
+        var questions = _context.Questions.Include(q => q.Subquestions)
```

Comment:
it would be more readable to use await here

Reply:
fixed